Installation/changes.html

```diff
@@ -148,6 +148,11 @@ <h2 id="release4.10">Release 4.10 </h2>
 <!-- Combinatorial Algorithms -->
 <!-- Geometry Kernels -->
 <!-- Convex Hull Algorithms -->
+  <h3>3D Convex Hull</h3>
+  <ul>
+    <li>The convex hull function can also produce a <code>Surface_mesh</code>,
+        and generally speaking any model of the concept <code>MutableFaceGraph</code></li>
+  </ul>
```

Comment:
The PR introduces five important changes, but only one is described in the `changes.html`. The other changes are important, and should also be described here.